packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -281,14 +278,6 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this._attachState = this.containerRuntime.attachState !== AttachState.Detached && this.existing ?
             this.containerRuntime.attachState : AttachState.Detached;
 
-        this.bindToContext = () => {
```

Comment:
Restore this, for compat tests.  But don't call it anywhere here. And continue to remove from interfaces.